### PR TITLE
Update nidcpower metadata

### PR DESCRIPTION
**Pull request by texasaggie97**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] ~~I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~
- [X] ~~I've added tests applicable for this pull request~~
### What does this Pull Request accomplish?
* Update nidcpower metadata
* For every property, adds documentation that specifies the corresponding LabVIEW API property
### What testing has been done?
* Unit
* System

### CODE REVIEW

File: docs/nidcpower/enums.rst [hunk 1]
```diff
@@ -91,6 +91,56 @@ ComplianceLimitSymmetry
 
 
 
+CurrentLevelAutorange
+---------------------
+
+.. py:class:: CurrentLevelAutorange
+
+    .. py:attribute:: CurrentLevelAutorange.OFF
```
Comment (marcoskirsch) [inline docs/nidcpower/enums.rst]: I think for these sort of attribute we decided to expose as Boolean in nimi-python.
I'm surprised they didn't call this NIDCPOWER_ATTR_CURRENT_LEVEL_AUTORANGE_ENABLED. I'll circle back to the team.

File: docs/nidcpower/enums.rst [hunk 2]
```diff
@@ -91,6 +91,56 @@ ComplianceLimitSymmetry
...
+        
+
+
+
+CurrentLimitAutorange
+---------------------
+
+.. py:class:: CurrentLimitAutorange
+
+    .. py:attribute:: CurrentLimitAutorange.OFF
```
Comment (marcoskirsch) [inline docs/nidcpower/enums.rst]: Should be Boolean

File: docs/nidcpower/enums.rst (hunk 1 above)
Comment (marcoskirsch) [reply docs/nidcpower/enums.rst]: Applies to all 4.

File: generated/nidcpower/_library.py [hunk 3]
```diff
@@ -311,14 +310,6 @@ def niDCPower_MeasureMultiple(self, vi, channel_name, voltage_measurements, curr
                 self.niDCPower_MeasureMultiple_cfunc.restype = ViStatus  # noqa: F405
         return self.niDCPower_MeasureMultiple_cfunc(vi, channel_name, voltage_measurements, current_measurements)
 
-    def niDCPower_ParseChannelCount(self, vi, channels_string, number_of_channels):  # noqa: N802
```
Comment (marcoskirsch) [inline generated/nidcpower/_library.py]: I thought we were generating this and using it as a private function, it's needed for figuring out memory allocation in niDCPower_MeasureMultiple()

File: generated/nidcpower/session.py [hunk 4]
```diff
@@ -306,14 +309,17 @@ class _SessionBase(object):
...
     '''
-    current_level_autorange = _attributes.AttributeViInt32(1150017)
+    current_level_autorange = _attributes.AttributeEnum(_attributes.AttributeViInt32, enums.CurrentLevelAutorange, 1150017)
     '''Type: bool
 
     Specifies whether NI-DCPower automatically selects the current level range based on the desired current level for  the specified channels.
-    If you set this property to AutoZero.ON, NI-DCPower ignores any changes you make to the  current_level_range property. If you change the current_level_autorange property from  AutoZero.ON to AutoZero.OFF, NI-DCPower retains the last value the current_level_range  property was set to (or the default value if the property was never set) and uses that value as the  current level range.
+    If you set this property to CurrentLevelAutorange.ON, NI-DCPower ignores any changes you make to the  current_level_range property. If you change the current_level_autorange property from  CurrentLevelAutorange.ON to CurrentLevelAutorange.OFF, NI-DCPower retains the last value the current_level_range  property was set to (or the default value if the property was never set) and uses that value as the  current level range.
```
Comment (marcoskirsch) [inline generated/nidcpower/session.py]: this change appears to be a mistake

File: src/nidcpower/metadata/attributes.py [hunk 5]
```diff
@@ -37,7 +37,7 @@
         'type': 'ViBoolean'
     },
     1050005: {
-        'access': 'read-write',
+        'access': 'read only',
```
Comment (marcoskirsch) [inline src/nidcpower/metadata/attributes.py]: I believe this is a mistake. Please double check internal NI meetadata (it was incorrect for a few days)

File: generated/nidcpower/_library.py (hunk 3 above)
Comment (texasaggie97) [reply generated/nidcpower/_library.py]: Fixed filtering in metadata generation

File: docs/nidcpower/enums.rst (hunk 1 above)
Comment (marcoskirsch) [reply docs/nidcpower/enums.rst]: We shipped that way? From our guidelines:

> Don't use enums when a bool will do
> C uses ViBoolean but provides named constants. Don't do that for Python
> LabVIEW APIs provide ring controls or enums. Don't do that for Python
> We think this is more obvious for Python clients

https://github.com/ni/nimi-python/wiki/API-design-guidelines#enums

File: README.rst [hunk 6]
```diff
@@ -72,7 +72,7 @@ NI-DCPower Python API Status
 +===============================+==========================+
 | Driver Version Tested Against | 18.1.1                   |
 +-------------------------------+--------------------------+
-| Driver API Version            | 19.1.0d33                |
+| Driver API Version            | 19.6.0d2                 |
```
Comment (marcoskirsch) [inline README.rst]: We should update from 19.1.0bX soon, since that will be released shortly.

**Pull request merged**